routes.go

```diff
@@ -56,6 +56,7 @@ func (h *Headscale) GetEnabledNodeRoutes(namespace string, nodeName string) ([]n
 	return routes, nil
 }
 
+// IsNodeRouteEnabled checks if a certain route has been enabled
 func (h *Headscale) IsNodeRouteEnabled(namespace string, nodeName string, routeStr string) bool {
```

Comment:
These functions should probably also be renames, but its probably not worth it.

I think the most important is that the database schema and models has the "correct" names so we dont have to do some major migration at some point.

Reply:
Fully agree. Machine and SharedMachine are now written in stone.